content/blog/2021/11/camel-k-release-1-7/index.md

```diff
@@ -0,0 +1,106 @@
+
+First of all, let's highlight the set of technologies on which Camel K 1.7 is based:
+
+* Apache Camel K Runtime 1.10.0
+* Apache Camel Quarkus 2.4.0
+* Apache Camel 3.12.0
+* Apache Camel Kamelets 0.5.0
+
+Thanks to Apache Camel, Camel Quarkus and Kamelet Catalog contributors for the great effor they've put in those new releases as well.
+
+## Dependencies upgrade
+
+The upgrades we applied in this new release, deserve a dedicate section.
+
+We upgraded `Golang` version to `1.6` which gave us the possibility to **bump several Kubernetes API** and so, be in line with the latest enhancements released in K8S space. We also moved to the latest `maven` version, which now point to `3.8.3`. We took the opportunity to promote `Knative` and `Service Binding` to their first `1.x` stable release and to move `Jolokia` to version `1.7.1`. You can find more details on the [GitHub release page](https://github.com/apache/camel-k/releases/tag/v1.7.0).
+
+## Quarkus native build support
+
+The most exciting and promising feature we brought in Camel K 1.7 is probably the support for `Quarkus` native build. If you're not yet aware of it, Quarkus is the great technology that allows your Java application to **run as fast as any native compiled application**. That helps reducing the resource footprint and reduce almost to zero the startup time, making your application a `Cloud Native` first class citizen.
+
+It was a long development and we're still polishing few things before adding official documentation (will do soon, keep tuned). You can find some useful information in the [discussion done during the development](https://github.com/apache/camel-k/pull/2536). The resume is that you can now launch your Camel application with a Quarkus Native profile:
+
+```
+$ kamel run -t quarkus.package-type=native ...
+```
+The build will need some time and some more resource than usual (average test of 5 minutes and 4GB of memory, but it will depend on each case) as there is an ahead of time compilation. The great news is that we thought that, while your native application is building, you can benefit the JVM mode as done normally so far:
+```
+$ kamel run -t quarkus.package-type=fast-jar -t quarkus.package-type=native ...
+```
+The Quarkus JVM application will be replaced with a rolling deployment update as soon as the native build will be completed!
+
+We will provide more official documentation soon. In the while, feel free to try it and report any problem, feedback or suggestion, we'd love to hear about how you're using it.
+
+This feature deserves a special mention to [Antonin Stefanutti](https://github.com/astefanutti) who has lead its development.
+
+## Installation and build improvements
+
+In the last release we've dedicated some time to include improvements about the way you can customize your Camel K installation on a Kubernetes cluster.
+
+### Kustomize
```

Comment:
I'd rather promote it as a top-level feature.

Reply:
That was more in order to simplify the reading part, as it belongs to *## Installation and build improvements* section. Both "### Kustomize" and "### Custom labels" are not meant to be downgraded as low level feature. If you still think it is better to keep at the same level of the rest, I'd remove the higher level section "## Installation and build improvements"

Reply:
The trouble with the current approach is that it somehow, granted involuntary, put _Kustomize_ and _Custom Labels_ at the same level, while IMHO, there are at completely different scales, both user and strategically wise. Actually, I almost wanted to add a review item to suggest to remove the "Custom labels" item. It's very low level and for very advanced users, given it requires a rebuild of the operator binary! Thinking about it, I don't think it's even intended for users, it's intended for downstream projects, that redistribute the Camel K operator. So a way to solve this all, is to remove the "Custom labels" section, and mechanically promote the "Kustomize" one.